jolpica/formula_one/data_correction.py

```diff
@@ -35,3 +40,22 @@ def run_data_correction():
         entry.is_eligible_for_points = False
         updated.append(entry)
     SessionEntry.objects.bulk_update(updated, fields=["position", "is_eligible_for_points"])
+
+
+def session_correction():
+    corrections = {
```

Comment:
Can you please add a 1 line comment to explain why this correct needs to be made (maybe reference its from issue #113 )

Reply:
Added a comment. The commit also contains a link to the original issue